topostats/validation.py

```diff
@@ -200,6 +200,28 @@ def validate_config(config: dict, schema: Schema, config_type: str) -> None:
                 "above",
                 error="Invalid direction for grains.direction valid values are 'both', 'below' or 'above",
             ),
+            "grain_height_removal_thresholds_std_dev": {
+                "above": [
+                    Or(
+                        float,
```

Comment:
Should the `float` be within a certain range? E.g. always `> 0.0`? I'm unsure of the expected ranges but see that the tests have arrays with negative values so `float` may be perfectly valid here but wanted to check as I thought all values are typically `> 0.0` (although perhaps some materials return negative values, I really don't know/remember).

Reply:
I think it would be sensible to set it so that it can only be positive. I don't see any reason why a negative value would be valid, as if you set the thresholds for above to be negative, that means that you're looking for grains detected above the image average, which are below the image average. It's nonsensical 👍. I have added lambda statements.